agent/agent.go

```diff
@@ -264,7 +263,8 @@ func initializeStateManager(cfg *config.Config, taskEngine engine.TaskEngine, cl
 		statemanager.AddSaveable("ContainerInstanceArn", containerInstanceArn),
 		statemanager.AddSaveable("Cluster", cluster),
 		statemanager.AddSaveable("EC2InstanceID", savedInstanceID),
-		statemanager.AddSaveable("ACSSeqNum", sequenceNumber),
+		//not tracking ACSSeqNum for now
```

Comment:
How about: "The `ACSSeqNum` field is retained for compatibility with `statemanager.EcsDataVersion` 4 and can be removed in the future with a version bump."  Alternately: just remove it now and bump the version?